### ISSUE TITLE
[BUG] The method "cui_default_pmig_properties" failed

### ISSUE THREAD

**Issue by AngelaIp** [ArasLabs/updated-properties-dialog#2]
## Expected Behavior
**How you expect the project to work?**
Property window should appear.

## Actual Behavior
**What is the project actually doing?**
Property window doesn´t start up at all. I just get an error message.

## Steps to Reproduce 
**How can we test this issue?**

1. Open an ItemType from TOC
2. Search and select Item in grid
3. Right click -> Properties -> Error messages appear
![image](https://user-images.githubusercontent.com/32091085/57512076-00658b00-730b-11e9-8fe6-65879db36c54.png)

When I start the properties dialog from an openend Item, nothing will happen.

### Aras Innovator 
* Major version: 11.0 
* Service pack(s): SP15 

### Browsers

- [ ] Internet Explorer 11
- [x] Firefox ESR 
- [x] Chrome 
- [ ] Edge 

### Project Version
**What version of this project are you using? Ex: v1.1.0**

### Additional Details

I followed the instructions and also modified the IncludeNamespaceConfig.xml. I also cleared the browser cache.

## Workaround / Solution
**Have you found a workaround or solution to this issue?**
Nope.

## Collaboration
**Are you willing to collaborate on this feature? Ex: sharing code, submitting a pull request, testing, etc.**
Why not.

**Comment by asebastian-aras** [comment]
Hi Angela,
I was unable to reproduce this issue. I just installed the project on a clean version of SP15. 

Could you verify that the ArasModules tag of the IncludeNamesapceConfig.xml file looks similar to the one below?
![image](https://user-images.githubusercontent.com/42644934/57533330-808bf080-730b-11e9-9b9e-5e973e680056.png)

Thanks,
AJ

**Comment by AngelaIp** [comment]
Hi AJ,

I used the exact same settings in the IncludeNamespaceConfig.xml. I made a few quick tests and was able to solve this one. But I could not determine the exact cause of this behaviour.

I temporary deleted this code part to test if the properties dialog will start up at all:
```
var code = new QRCode(document.getElementById("qrCode"), {
			text: text,
			width: 192,
			height: 192,
			colorDark : "#000000",
			colorLight : "#ffffff",
			correctLevel : QRCode.CorrectLevel.H
		});
		document.getElementById("qrCode").hidden = true;
```
Without this code, the properties dialog worked - of course without qrcode.
I than retyped the quotation marks around "qrCode" and uncommented the code block again. 
And suddenly it worked!

I tested this one in a SP15 test environment. I will later add this project to the main instance. I will let you know if I see the same behaviour again. I assume my download package was corrupted. 

Thanks for your help!
Angela
